content/blog/scverse_announcement.md

```diff
@@ -0,0 +1,41 @@
+
+Our primary experience lies in the field of computational genomics and lately single-cell sequencing technologies, and this has defined our initial scope and focus in the very beginning of our journey. We are excited to announce **scverse** a consortium for joint maintenance of core packages in omics data analysis in life sciences.
+
+## Who is a part of scverse?
+
+scverse is explicitly an effort beyond just a single research group. Only together can we continue to ensure that the ecosystem grows and stays interoperable.
+We therefore define several roles of which only the most relevant are denoted here:
+
+**Isaac Virshup**, **Danila Bredikhin** and **Lukas Heumos**, who are also a part of the steering committee, belong to the core developers together with **Adam Gayoso**, **Giovanni Palla**, **Gregor Sturm**, and **Ilia Kats**. The core developers have made significant contributions to the ecosystem and are jointly developing and maintaining the major libraries.
+
+Our management committee, consisting of **Fabian Theis**, **Oliver Stegle**, **Nir Yosef**, **Alex Wolf** and **Francesca Finotello**, are providing resources and academic freedom to scverse to develop our tools.
+
+**Aviv Regev**, **Sarah Teichmann**, **Dana Pe'er** and **Bonnie Berger** are our scientific advisory board which defines the overall vision and determines the most important needs.
+
+## What tools does scverse maintain?
+
+We start with the two most important data structures, [AnnData](https://github.com/scverse/anndata) and [MuData](https://github.com/scverse/mudata), together with [Scanpy](https://github.com/scverse/scanpy) for general purpose single-cell analysis, [Muon](https://github.com/scverse/muon) for multimodal single-cell analysis, [scvi-tools](https://github.com/scverse/scvi-tools) for deep probabilistic analysis of single-cell data and [scirpy](https://github.com/scverse/scirpy) for T-cell receptor analysis. We are planning to expand very soon and are actively looking for major libraries and frameworks which fit our vision.
+
+## What are we doing?
+
+While we're continuing to work on the scverse libraries and frameworks, we're trying to improve how they work and are tested together. Hence, we are looking into the development of infrastructure which cross tests release candidates of our tools. Further, we are building tooling to more easily generate scverse based packages and analyses to grow the ecosystem. Beyond technical challenges we are looking to build a community around scverse with for example hackathons and to teach the community our tools with workshops.
```

Comment:
I would start with community, and how establishing community will help interoperability/technical challenges

Reply:
we also have goals related to accessibility of the packages, reducing barrier to entry of usage and development.